### PR TITLE
tests/unittests/tests-tweetnacl: increase needed stack

### PR DESCRIPTION
**Pull request by kYc0o**
See #6436

### CODE REVIEW

**Comment by kYc0o** [comment]
Tested in mulle:

```console
2017-01-26 14:14:24,027 - INFO # ��main(): This is RIOT! (Version: 2016.10-devel-1455-ga83a4-snake.local-tweetnac_inc_stack)
2017-01-26 14:14:30,987 - INFO # ............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
2017-01-26 14:14:30,988 - INFO # OK (716 tests)
```

File: tests/unittests/tests-tweetnacl/Makefile.include [hunk 1]
```diff
@@ -1,2 +1,4 @@
 USEMODULE += random
 USEPKG += tweetnacl
+
+CFLAGS += -DTHREAD_STACKSIZE_MAIN=\(2*THREAD_STACKSIZE_DEFAULT+THREAD_EXTRA_STACKSIZE_PRINTF\)
```
Comment (gebart) [inline tests/unittests/tests-tweetnacl/Makefile.include]: missing trailing newline

**Comment by kYc0o** [comment]
Comments addressed and fixed some errors spotted by Jenkins, immediately squashed.

**Comment by kYc0o** [comment]
Uhm... it seems I cannot exclude some boards of compiling tweetnacl test, I'll need to add some boards to the insufficient memory list...

**Comment by OlegHahm** [comment]
Does this fix #6436?

(Pro tip: if you write "fixes #6436 in the PR description, Github will automatically close the issue once the PR is merged.)

**Comment by kYc0o** [comment]
Yes it does. I'll think about that the next time, thanks Oleg!

**Pull request merged**